### PR TITLE
Additional CS courses from Cornell.

### PR DESCRIPTION
**Pull request by awelch83**
@prakhar1989 To answer your concerns from #93, I agree that courses which offer assignments and lectures, in addition to readings, are more useful. I still think, however, that CS 6650 serves a purpose, because this subject matter isn't currently addressed by any other course listed in the readme. In the future, if someone finds a course on this subject that does have assignments to go with it, they will hopefully delete my entry and replace it with their own. Aside from that, I also got rid of the extra bullet point that I somehow created at line 615, in pr #93.

@robbyoconnor I deleted .DS_Store, as you mentioned. I'm not sure why, or how, that file somehow finds its way into my commits from time to time.

### CODE REVIEW

**Comment by ahmadassaf** [comment]
.DS_Store "(short for Desktop Services Store)[1] is a hidden file with a proprietary format created by OS X to store custom attributes of a folder such as the position of icons or the choice of a background image"  [Wikipedia](https://en.wikipedia.org/wiki/.DS_Store)

Bottomline, they will keep coming back because they are generated automatically by the OS. To avoid that, you should use  `gitignore` files. This [link](http://git-scm.com/docs/gitignore) will help you overcome this issue.

File: README.md [hunk 1]
```diff
@@ -179,17 +214,26 @@ Courses
...
 	- [Lectures](http://www-scf.usc.edu/~csci104/20142/lectures)
 	- [Labs](http://www-scf.usc.edu/~csci104/20142/labs)
 	- [Assignments](http://www-scf.usc.edu/~csci104/20142/assignments/)
 	- [Additional Resources](http://www-scf.usc.edu/~csci104/20142/resources.html)
+- [CS 4820](http://www.cs.cornell.edu/courses/CS4820/2014sp/) **Introduction to Analysis of Algorithms** *Cornell University* <img src="https://assets-cdn.github.com/images/icons/emoji/unicode/1f4bb.png" width="20" height="20" alt="Assignments" title="Assignments" /> <img src="https://assets-cdn.github.com/images/icons/emoji/unicode/1f4dd.png" width="20" height="20" alt="Lecture Notes" title="Lecture Notes" /> <img src="https://assets-cdn.github.com/images/icons/emoji/unicode/1f4da.png" width="20" height="20" alt="Readings" title="Readings" />
```
Comment (prakhar1989) [inline README.md]: @awelch83 Isn't this course added twice? line: 217?

**Comment by awelch83** [comment]
@prakhar1989 Problem fixed. There was no duplicate of CSCI 104, but I did find a duplicated of CS 4820, which I removed.

**Comment by awelch83** [comment]
Thanks @ahmadassaf. I'm going to read about gitignore later today.

**Comment by prakhar1989** [comment]
Thanks! Merging this now :grinning:

**Pull request merged**